dev-python/shiboken/shiboken-5.12.3.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+CMAKE_MIN_VERSION=3.1.0
+
+inherit cmake-utils llvm python-r1
+
+MY_P="pyside-setup-everywhere-src-${PV}"
+
+DESCRIPTION="Python binding generator for C++ libraries"
+HOMEPAGE="https://wiki.qt.io/PySide2"
+SRC_URI="https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-${PV}-src/${MY_P}.tar.xz"
+
+# The "sources/shiboken2/libshiboken" directory is triple-licensed under the
+# GPL v2, v3+, and LGPL v3. All remaining files are licensed under the GPL v3
+# with version 1.0 of a Qt-specific exception enabling shiboken2 output to be
+# arbitrarily relicensed. (TODO)
+LICENSE="|| ( GPL-2 GPL-3+ LGPL-3 ) GPL-3"
+SLOT="2"
+KEYWORDS="~amd64 ~x86"
+IUSE="+docstring numpy test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+BDEPEND=">=sys-devel/clang-6.0.0:="
+RDEPEND="${PYTHON_DEPS}
+	>=dev-qt/qtcore-${PV}:5
+	docstring? (
+		>=dev-libs/libxml2-2.6.32
+		>=dev-libs/libxslt-1.1.19
+		>=dev-qt/qtxml-${PV}:5
+		>=dev-qt/qtxmlpatterns-${PV}:5
+	)
+	numpy? ( dev-python/numpy[${PYTHON_USEDEP}] )
+"
+DEPEND="${RDEPEND}
+	test? ( >=dev-qt/qttest-${PV}:5 )
+"
+
+S="${WORKDIR}/${MY_P}/sources/shiboken2"
+DOCS=( AUTHORS )
+
+# Ensure the path returned by get_llvm_prefix() contains clang as well.
+llvm_check_deps() {
+	has_version "sys-devel/clang:${LLVM_SLOT}"
+}
+
+src_prepare() {
+	#FIXME: File an upstream issue requesting a sane way to disable NumPy support.
+	if ! use numpy; then
+		sed -i -e '/print(os\.path\.realpath(numpy))/d' \
+			libshiboken/CMakeLists.txt || die
+	fi
+
+	# CMakeLists.txt assumes clang builtin includes are installed under
+	# LLVM_INSTALL_DIR. They are not on Gentoo. See bug 624682.
+	sed -i -e "s~clangPathLibDir = findClangLibDir()~clangPathLibDir = QLatin1String(\"${EPREFIX}/usr/lib\")~" \
```

Comment:
(minor) this should be using `QStringLiteral` rather than `QLatin1String`

Reply:
**Excellent.** I stole this one-liner wholesale from @Perlovka and only have a dim idea of how and why it works. This is why you're the Qt engineer in the house.